samples/config/config.sample

```diff
@@ -71,3 +71,34 @@
 # Keys:
 #   camtopic
 #      Gazebo topic where camera images are published.
+#
+#
+# Section [imgcap]:
+#
+# Keys:
+#   width
+#       Width of the image to be captured in Pixels.
+#       Default: Width of the camera sensor.
+#       Possible Values:
```

Comment:
I presume here you plan on having a max and min range? In any case, this is empty ...

Reply:
This is also dependent on camera sensor, cannot fix it. I can state `Dependent on camera sensor` that instead of leaving empty

Reply:
Fixed.